packages/eslint-plugin/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.3.0](https://github.com/endojs/endo/compare/@endo/eslint-plugin@2.2.3...@endo/eslint-plugin@2.3.0) (2025-01-23)
+
+
+### Features
+
+* **ses:** Add XS variant of shim ([f6c8456](https://github.com/endojs/endo/commit/f6c84566bb6a698709dc3474726000f07b94f3db))
```

Comment:
That PR probably shouldn't have been squashed to avoid a minor version bump of eslint-plugin

Reply:
In this case, I simply didn’t have the foresight to separate the eslint change into its own conventional commit. I’ll have to watch for that in the future.